src/pages-and-resources/discussions/app-config-form/utils.js

```diff
@@ -27,14 +26,34 @@ export const checkStatus = ([startDate, endDate]) => {
   return status;
 };
 
-export const formatDate = (date, time) => (time ? `${date}T${time}` : date);
+export const stringifyDateTime = (date, time) => {
```

Comment:
I think `date` & `time` are already strings, `stringifyDateTime` doesn't exactly convey the responsibility of this fn.